Update what-are-analysis-environments.md (#2640)
enkidevs/curriculum: python-data-analysis/python-data-analysis-core/analysis-environments/what-are-analysis-environments.md

When it comes to coding in Python, there are several environments you can choose from.

These environments are shells, [IDE's](https://www.enki.com/glossary/general/ide), and other ways where you can test, run, modify, execute, and so on, a programming language's code.

For instance, the most basic environment you can write Python in is your OS's terminal[1]. 

To open the Python environment in your terminal (known as *initializing the [interpreter](https://www.enki.com/glossary/general/interpreter)*), type in the word `python` and press Enter.

> 💡 Different versions of Python will have different keywords to launch the interpreter. The most common ones are `py` and `python`.

> Even though most operating systems come with a pre-installed version of the Python interpreter it is not the recommended tool for codinng as it lacks many features.

Author: ```suggestion
Even though most operating systems come with a pre-installed version of the Python interpreter it is not the recommended tool for coding as it lacks many features.
```

> There are far superior environments with better interpreters for coding in Python, such as

Author: ```suggestion
There are far superior environments with better interpreters and more powerful features, such as:
```
